ceno_zkvm/src/scheme/verifier.rs

```diff
@@ -245,10 +245,13 @@ impl<E: ExtensionField, PCS: PolynomialCommitmentScheme<E>> ZKVMVerifier<E, PCS>
                     &challenges,
                 )?;
                 rt_points.push(input_opening_point);
-                evaluations.push(table_proof.wits_in_evals.clone());
-                if circuit_vk.cs.num_fixed() > 0 {
-                    evaluations.push(table_proof.fixed_in_evals.clone());
-                }
+                evaluations.push(
```

Comment:
same as respective prover part

Reply:
I think this also have few impact on the verifier's performance. Since it's more easier to read, I prefer to keep the current codes.